src/scene/materials/standard-material-validator.js

```diff
@@ -90,7 +90,7 @@ class StandardMaterialValidator {
                 if (REMOVED[key]) {
                     delete data[key];
                 } else {
-                    Debug.warn(`Ignoring unsupported input property to standard material: ${key}`);
+                    Debug.warnOnce(`Ignoring unsupported input property to standard material: ${key}`);
```

Comment:
were you going to add material name?

Reply:
I looked at it, but we don't have the material there handy, and I don't really want to import it just for a debug log.
Happy to close this PR as well though, now that you've fixed the logging.